### PR TITLE
fix(dropdown): fix dropdown button click

### PR DESCRIPTION
**Pull request by RuanRamalho**
[#595 Issue](https://github.com/gympass/yoga/issues/595)

## Description 📄

Hello guys! 

To solve the problem with the `dropdown` `button` click, it was necessary to assign a `padding bottom` of sixteen pixels to the `button` component, so that the user can execute its functionality when clicking on the bottom part of the `button`

**For more details access the issue [#595](https://github.com/gympass/yoga/issues/595)**

## Platforms 📲

- [x] Web
- [ ] Mobile

## Type of change 🔍

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? 🧪

[Enter the tips to test this PR]

- [ ] Unit Test
- [x] Snapshot Test

## Checklist: 🔍

- [x] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [ ] Layout matches design prototype: [FIGMA](https://figma.com/file/YOUR_LINK)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

## Screenshots 📸

| Before | After |
| ------ | ----- |
|    image    |   image    |

### CODE REVIEW

**Comment by matheushdsbr** [comment]
Yo @RuanRamalho! First of all, congratulations for your contribution! 🚀 🚀 🚀 

Thank you for noticing this bug and taking this initiative with us, your contribution is certainly making a difference to the evolution of Yoga o/

**Review by caiotracera — APPROVED** [review]
Hey, @RuanRamalho!
First of all, congrats on your first PR here on Yoga!

The code LGTM!

File: packages/yoga/src/Dropdown/web/Dropdown.jsx [hunk 1]
```diff
@@ -141,6 +141,7 @@ const Button = styled.button`
     width: 100%;
     padding-right: ${dropdown.button.padding.right}px;
     padding-top: ${dropdown.button.padding.top}px;
+    padding-bottom: ${dropdown.button.padding.top}px;
```
Comment (caioalexandrebr) [inline packages/yoga/src/Dropdown/web/Dropdown.jsx:144]: I'm happy for your first contribution!! 🚀
Thanks for the contribution.

I saw that you created the bottom property in the theme file, how about using it here?

```suggestion
    padding-bottom: ${dropdown.button.padding.bottom}px;
```

**Review by caioalexandrebr — COMMENTED** [review]
Does the native Dropdown version suffer from the same problem?

**Review by caioalexandrebr — APPROVED** [review]

**Review by matheushdsbr — APPROVED** [review]

**Comment by RuanRamalho** [comment]
Thanks so much for reviewing my code, I am very happy!!

**Comment by matheushdsbr** [comment]
@all-contributors please add @RuanRamalho for bug

**Pull request merged**